front_end/generated/protocol-proxy-api.d.ts

```diff
@@ -3053,6 +3053,8 @@ declare namespace ProtocolProxyApi {
   }
 
   export interface TracingApi {
+    invoke_getTraceSystemsAndLevels(): Promise<Protocol.Tracing.GetTraceSystemsAndLevelsResponse>;
```

Comment:
```suggestion
    // COHERENT BEGIN
    invoke_getTraceSystemsAndLevels(): Promise<Protocol.Tracing.GetTraceSystemsAndLevelsResponse>;
    // COHERENT END
```

Reply:
this is a generated file. All `COHERENT BEGIN` / `COHERENT END` comments for changes responsible for this are in `browser_protocol.pdl`